templates/iot/examples/iot-config.yaml

```diff
@@ -5,8 +5,8 @@ metadata:
 spec:
   services:
     deviceRegistry:
-      file:
-        numberOfDevicesPerTenant: 100000
+      infinispan:
+        infinispanServerAddress: example-infinispan:11222
```

Comment:
Is there a way to extract this value from Infinispan service?

Reply:
At the moment we assume the user will deploy infinispan on his own, hence putting the correct value in here. WDYT ?